ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.ts

```diff
@@ -8,4 +9,5 @@ import { Component, Input } from '@angular/core';
 export class FaqSectionComponent {
     @Input() questions: string;
     @Input() questionsCount: number; /* don't pass the counter for displaying all questions */
+    @ViewChild(MatAccordion) accordion: MatAccordion;
```

Comment:
#minor I'd recommend to add a comment that this public property is used outside of component in order to prevent other developers desire to clean it up :)

Reply:
Put a petition comment there (7c8055f26248f550dfcdc8e9260f6caf1ca2a76d)